challenges/shakespearian-insult-generator.js

```diff
@@ -17,3 +17,11 @@ var second_word = ["weather-bitten", "unchin-snouted", "toad-spotted", "tickle-b
 var third_word = ["wagtail", "whey-face", "vassal", "varlet", "strumpet", "skainsmate", "scut", "ratsbane", "pumpion", "puttock", "pignut", "pigeon-egg", "nut-hook", "mumble-news", "moldwarp", "miscreant", "minnow", "measle", "mammet", "malt-worm", "maggot-pie", "lout", "lewdster", "joithead", "hugger-mugger", "horn-beast", "hedge-pig", "harpy", "haggard", "gudgeon", "giglet", "fustilarian", "foot-licker", "flirt-gill", "flax-wench", "flap-dragon", "dewberry", "death-token", "codpiece", "coxcomb", "clotpole", "clack-dish", "canker-blossom", "bum-bailey", "bugbear", "boar-pig", "bladder", "barnacle", "baggage", "apple-john"];
 
 // YOUR CODE HERE
+
+
+function insultGen() {
+  let one = first_word[parseInt(Math.random() * first_word.length)];
+  let two = second_word[parseInt(Math.random() * second_word.length)];
+  let three = third_word[parseInt(Math.random() * third_word.length)];
+  return ()"You " + one + ", " + two + ", " + three + "you!");
```

Comment:
I'm confused about your parentheses on this line, but otherwise this code looks good!

Reply:
I see the extra parenthesesnow, Thanks for catching it!